.github/workflows/sign-darwin.yml

```diff
@@ -0,0 +1,61 @@
+---
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: 'release tag to sign'
+        required: false
+        type: string
+
+jobs:
```

Comment:
let's move this whole thing to `release-main.yaml`

Reply:
So this requires inputs to work, we can and maybe should call these on another workflow like that found in the release-main process. At least that was my hope. Here at least this allows us a repeatable mechanism to get this to work.

Reply:
The inputs can be populated by the release-please task outputs the `bump-kib` job does this. 

Keeping this here while it's draft is fine, but I really want to see it in the other file as another workflow.

Reply:
Sure thing. There were some difficulties in the release job process itself that kept me from moving forward to that approach, it has been a while but I tried to communicate my frustration with that approach.